gnucash/gnucash-310.css

```diff
@@ -27,7 +27,7 @@
   padding: 2px 2px 2px 2px; /* all work with different values, around the text blue area */
 }
 
-.cursor .button {
+.cursor .toggle-button {
```

Comment:
I'd prefer not to do this. Granted the new version is lighter. On the flip side though it's also no longer similar to other arrow buttons used in for example a combobox. So it is slightly alienating the register in a gnome widget context.

I would propose to try and use more vertical space per line so the toggle button gets slightly more room. This will make the register less compact, but it's no longer a pixel perfect match with the old register anyway.

This may also solve a few other glitches which I mention here (but are due to other code sections):
- If text in the cell cursor as glyphs that drop below the base line (like 'g', 'j',...) these glyphs glue against the double line. This is probably because you moved this double line inside the cell again. And perhaps the theme has something to do with it as well.
- Similarly if I click in a cell having such glyphs, the lower line of the glyph is gone (overwritten with yellow somewhere).
- The bottom padding/margin is not correct when using  a huge font. There also the bottom part of similar glyphs is gone. Note the current situation is already a big improvement compared to before your changes. Before the lines wouldn't even scale at all and I saw as much as half of the text being cut off vertically.